modules/administration-guide/pages/specifying-the-update-approval-strategy.adoc

```diff
@@ -0,0 +1,38 @@
+:_content-type: ASSEMBLY
+:navtitle: Specifying the update approval strategy
+:keywords: administration guide, upgrading-che-using-operatorhub
+:page-aliases: installation-guide:upgrading-che-using-operatorhub
+
+[id="specifying-the-{prod-id-short}-operator-update-approval-strategy-using-the web-console_{context}"]
+= Specifying the {prod-short} operator update approval strategy using the web console
+
+
+The {prod-short} operator supports two upgrade strategies:
+
+`Automatic`::
+The Operator installs new updates when they become available.
+
+`Manual`::
+New updates need to be manually approved before installation begins.
```

Comment:
```suggestion
You approve the updates before the operator installs them.
```
Guideline: Avoid passive voice.
- Usually, words like "manually" and "automatically" are red flags that a phrase doesn't identify the person or thing that performs the action. If you identify the person or thing, those words are almost always unnecessary. 
- Other red flag words: is, are, to be

Reply:
That's actually a copy-paste from the OpenShift 4.10 interface, therefore I will keep it.